sdks/python/src/opik/api_objects/helpers.py

```diff
@@ -22,3 +30,35 @@ def datetime_to_iso8601_if_not_None(
 
 def list_to_batches(items: List[Any], batch_size: int) -> List[List[Any]]:
     return [items[i : i + batch_size] for i in range(0, len(items), batch_size)]
+
+
+def get_project_name(
```

Comment:
`get` is probably too simple for such function, it should be named something like "resolve_child_span_project_name"